docs/admin/releases/14/README.rst

```diff
@@ -0,0 +1,178 @@
+- Admins can now set custom landing pages for students and teachers with the "student_home_page" and "teacher_home_page" tags, respectively
+- Added buttons to the custom form response interface to bulk download files
+- Classes are now colored by category in the popularity graphs on the student big board
+- Tweaked the tooltip of the popularity graphs on the student big board
+- Student and teacher registration status is now shown on the userview page
+- Fixed the color of messages on the deadline management page
+- Added statistics to the dashboard for "teachers who have submitted a class and have not taught for a program" and "students who are enrolled and have not enrolled in the past"
+- Added the ability to edit existing teacher events (e.g., trainings, interviews) on the teacher event page
+- Fixed running the phase zero lottery with complex grade caps (e.g., a single capacity across multiple grades)
+- Added error messages to the phase zero lottery page if the grade cap tag is not set properly
+- Fixed the role name of lottery winners (removed the "s")
+- Clarified the help text for the "Priority limit" setting and removed the "Use priority" setting in the student registration settings
+- Uploaded documents can now be renamed
+- Made several fixes and enhancements to the lunch constraints form (e.g., fixed the initial values of the form, lunches are no longer deleted if the form is resubmitted)
+
+Class and section status management
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- "Rejection" can now only happen before a class is scheduled, whereas "cancellation" can now only happen after a class is scheduled (this change is now implemented on the class management page,  dashboard, and class search page)
+- When a section's status is changed, if all sections of a class now have the same status, the status of the class is changed to that status
+- Section management forms now are submitted individually
+- Sections and classes can now only be cancelled via the cancellation form
+- Sections can no longer be approved unless their parent class is also approved
+
+Death of the admin pages
+~~~~~~~~~~~~~~~~~~~~~~~~
+- Added a user interface for adding and editing class categories and class flag types (/manage/categoriesandflags)
+- Added a user interface to approve and reject grade change requests on the userview page
+- Added a user interface to create, edit, and delete permissions for individual users (on what was previously the Deadline Management page)
+- Added a user interface to create, edit, and delete URL and email redirects (/manage/redirects)
+- Added a user interface to edit registration receipts on the program settings page
+- Added a user interface to change the sequence and requiredness of program modules
+- Added a user interface for adding and removing students to/from the student lottery
+- The link to the admin pages has been removed for all themes
+
```

Comment:
Are there links to these new /manage/* pages anywhere (such as from the QSD textbox on /manage/programs/ page?

Reply:
Of course! (and that isn't a QSD box anymore)
![image](https://user-images.githubusercontent.com/7232514/174623033-0c381e41-8aae-4008-bb09-31b0aa99ca79.png)